tests/account-management.spec.ts

```diff
@@ -0,0 +1,29 @@
+// import { test, expect } from '@playwright/test';
```

Comment:
Is this still a WIP?

Reply:
this test runs fine locally when mailcatcher is also running, but fails in CI probably due to this: https://support.circleci.com/hc/en-us/articles/360007444314-Troubleshooting-sending-email-s-from-a-container

I think there's still value in being able to run this test locally, so will likely gate it behind a `process.env.IS_CI` check or something